pkg/apis/postgres-operator.crunchydata.com/v1beta1/shared_types.go

```diff
@@ -333,6 +337,13 @@ type AdditionalVolume struct {
 	// +optional
 	Containers []DNS1123Label `json:"containers"`
 
+	// Details for adding an image volume
+	// ---
+	// https://kubernetes.io/docs/concepts/storage/volumes/#image
```

Comment:
⛏️ I like to shorten these a little.


```suggestion
	// https://docs.k8s.io/concepts/storage/volumes#image
```